src/children.js

```diff
@@ -20,8 +20,7 @@ export default function children (node) {
 
 	if (childProperties) {
 		return childProperties.flatMap((property) => node[property] ?? []);
-	}
-	else {
+	} else {
```

Comment:
What happened to linting here? The previous one was correct. Is the ESLint file not applied in the new repo?